content/en/service_management/workflows/private_actions/use_private_actions.md

```diff
@@ -76,6 +76,21 @@ From the **Private Action Runner** page in [Workflow Automation][6] or [App Buil
    You can safely ignore the error `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
 
 [101]: https://docs.docker.com/compose/compose-application-model/
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
```

Comment:
Note for reviewer: this doc has the same stuff three times. The three copies of this new info are identical.